veloroutes_voies_vertes/processing/covadis/export_covadis.py

```diff
@@ -184,7 +184,6 @@ def toSHP(self, context, table, dpt, dirname, uri, geom, charger):
             transformContext=transformContext,
             options=options
         )
-        print(error)
         if error[0] == QgsVectorFileWriter.ErrCreateLayer:
```

Comment:
```suggestion

        # Vérification du retour de l'enregistrement et des erreurs potentiels
        if error[0] == QgsVectorFileWriter.ErrCreateLayer:
```
Ajout d'une ligne et d'un commentaire.